src/apps/Map/components/Brace.js

```diff
@@ -1,27 +1,61 @@
 import React from 'react';
 import styled from 'styled-components';
 
-// import BraceHorizontalZustand from '~/images/brace-horizontal-zustand.svg';
-// import BraceHorizontalPlanung from '~/images/brace-horizontal-planung.svg';
+import BraceHorizontal from '~/images/selector-brace.svg';
+import BraceVertical from '~/images/selector-brace_vertical.svg';
+import { media } from '~/styles/utils';
 
-import BraceHorizontalPlanung from '~/images/planung-brace.svg';
-import BraceHorizontalZustand from '~/images/zustand-brace.svg';
-
-const StyledBraceHorizontalZustand = styled(BraceHorizontalZustand)`
+const StyledBraceHorizontalZustand = styled(BraceHorizontal)`
   display: block;
   width: 100%;
-  margin: 0.5em 0;
+  margin-left: -30px;
+
+  ${media.m`
+    display: none;
+  `}
 `;
 
-const StyledBraceHorizontalPlanung = styled(BraceHorizontalPlanung)`
+const StyledBraceHorizontalPlanung = styled(BraceHorizontal)`
   display: block;
   width: 100%;
-  margin: 0.5em 0;
+  margin-left: 30px;
+
+  ${media.m`
+    display: none;
+  `}
+`;
+
+const StyledBraceWrapper = styled.div`
+  display: block;
+  // This is the width of the SVG file
+  width: 950px;
+  position: absolute;
+  left: 50%;
+  margin-left: -475px;
+  overflow: hidden;
+
+  ${media.m`
+    width: 100%;
+    height: 200px;
+    position: static;
+    margin: -50px 0;
+  `}
+`;
+
+const StyledBraceVertical = styled(BraceVertical)`
+  display: block;
+  height: 100%;
 `;
 
-export default (props) =>
-  props.type === 'zustand' ? (
-    <StyledBraceHorizontalZustand />
-  ) : (
-    <StyledBraceHorizontalPlanung />
+export default (props) => {
+  return (
+    <StyledBraceWrapper>
+      {props.type === 'zustand' ? (
+        <StyledBraceHorizontalZustand />
+      ) : (
+        <StyledBraceHorizontalPlanung />
+      )}
```

Comment:
I found those two Horizontal Styled Components hard to read. I think you could just use one and provide the `30px` vs. `-30px ` as props. The following code will not work, yet (where to I tell the `styled` to accept my custom prop?; did not look it up) but just to illustrate it would make it a lot shorter.

```jsx
const StyledBraceHorizontal = styled(BraceHorizontal)`
  display: block;
  width: 100%;
  margin: 0.5em 0;
  margin-left: ${pointerToButtonAdjustment};
  ${media.m`
    display: none;
  `}
`;
```
```suggestion
      <StyledBraceHorizontal pointerToButtonAdjustment='30px' />
```

Reply:
You're right, I corrected it with that pattern!